spec/support/devise_macros.rb

```diff
@@ -0,0 +1,10 @@
+# Spec helpers for Devise Token auth
+module DeviseMacros
```

Comment:
@chrisdolendo yeah both Devise and Token Auth each give instructions of how to set up tests (and I got part of this from some comments in a thread on Token Auth) but they don't provide anything specific for some of this stuff since you could be using different test suites, etc... just some examples. Also most "login" examples are for controller tests (not request tests) so it took me a while to get this working and get it right 😌